internal/trace/transport.go

```diff
@@ -87,3 +93,69 @@ func logHeader(header http.Header) string {
 	}
 	return "   Empty header"
 }
+
+// logResponseBody prints out the response body if it is printable and within
+// the size limit.
+func logResponseBody(resp *http.Response) string {
```

Comment:
Printing the response body leads to security issues. Basically, it prints out the token, which it should not.

![image](https://github.com/user-attachments/assets/76d5c3f2-116c-4a8f-8f7e-3df134cfb58a)

/cc @FeynmanZhou

Reply:
We can start from hiding the body if it contains keywords "token" and "access_token".